### PR TITLE
fix: #723 fixed mistake with Add tag button

### PR DESCRIPTION
**Pull request by AKiroi**
resolved problem in this task https://github.com/tritonJS826/masters-way/issues/723

https://github.com/tritonJS826/masters-way/assets/112805668/a4f92256-3639-44c6-ad15-26c04ce68c1c

### CODE REVIEW

**Review by tritonJS826 — APPROVED** [review]

**Comment by tritonJS826** [comment]
https://github.com/tritonJS826/masters-way/assets/56690686/64338711-888f-48a3-915b-cee4360989d3

File: mw-webapp/src/logic/wayPage/labels/AdjustLabelsModalContent.tsx [hunk 1]
```diff
@@ -106,24 +106,27 @@ export const AdjustLabelsBlock = (props: JobTagsProps) => {
...
-            />
-          }
-          trigger={
-            <Button
-              value={LanguageService.way.filterBlock.addTagButton[language]}
-              onClick={() => setIsJobDoneModalOpen(true)}
-            />
-          }
-        />}
+        <div className={styles.wrapperButton}>
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/labels/AdjustLabelsModalContent.tsx:109]: 1. Bad naming, if name wrapperButton this div should be around button but not around all modal
2. I thimk that better to add this class inside modal component for get rid the same situation in the future

@tritonJS826  What do you think?

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]
Wrote comment

File: mw-webapp/src/logic/wayPage/labels/AdjustLabelsModalContent.tsx (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/labels/AdjustLabelsModalContent.tsx]: fixed

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**